src/components/Auth.vue

```diff
@@ -1,38 +1,44 @@
 <template>
-  <v-layout row align-baseline>
-    <v-btn v-if = "!loggedIn" outline round color = "primary" @click="loginUser">
+  <v-layout row align-center>
+    <v-btn v-if="!loggedIn" outline round color="primary" @click="loginUser">
```

Comment:
Why remove all the spaces around `=`s? Honestly I have no idea what standard linters do but I always thought space around assignment `=` signs was easier to read.

Reply:
I'm not sure what standard linters do either, but the examples [here](https://vuejs.org/v2/style-guide/) don't use the spaces. A close parallel I'm familiar with is props for React components, which don't use the spaces either (`<Blah prop={x} />`).

I personally removed them because my code editor doesn't do syntax highlighting correctly for the stuff inside the `""` unless those spaces aren't there.

Reply:
Ah, I agree that normal assignment operators shouldn't have them though. I'll go through and try to reverse those cases.

Reply:
Oh, yeah, I left this comment in a bad place. I totally agree for things like this, because they're like keyword arguments. Should have chosen an assignment line

Reply:
And also a lot of conditionals have `a===b`, which I think is nonstandard

Reply:
lgtm (commit 659189e)

Reply:
oh, no, there are more, I thought it would be a quick regex match thing but there were a lot that didn't pick up and I'm trying to go to bed :P

Reply:
oops :disappointed_relieved:

Reply:
it looks like a lot of it happened in this commit: b6d9fe5ebcad07c60ef92fe778d2ef9f93e18838

so you might be able to just rebase it out. there were some other good changes in there though so it might be a pain either way